R/expected_time.R

```diff
@@ -109,7 +109,7 @@ expected_time <- function(enroll_rate = tibble::tibble(
   #    build a help function    #
   # ----------------------------#
   # find the difference between  `AHR()` and different values of total_duration
-  foo <- function(x) {
+  ahr_diff <- function(x) {
```

Comment:
shall we call it `event_diff`? Since basically it calculates the difference between the targeted events and the events at a calendar time...

Reply:
sure updated.